### ISSUE TITLE
ParseException when @SuppressWarnings in for statement

### ISSUE THREAD

**Issue by axel3rd** [cobertura/cobertura#109]
Cobertura 2.0.3

This snippet code :
```
Collection<String> c = new ArrayList<String>();
for (@SuppressWarnings("unchecked")
	Iterator<String> it = c.iterator(); it.hasNext();) {
	System.out.println(it.next());
}
```

Crashes with exception : 
```
net.sourceforge.cobertura.javancss.parser.ParseException: Encountered " "@" "@ "" at line XXX, column 22.
Was expecting one of:
    "assert" ...
    [...]
    "double" ...
    
	at net.sourceforge.cobertura.javancss.parser.JavaParser.generateParseException(JavaParser.java:11869)
	at net.sourceforge.cobertura.javancss.parser.JavaParser.jj_consume_token(JavaParser.java:11730)
	at net.sourceforge.cobertura.javancss.parser.JavaParser.ForInit(JavaParser.java:4409)
	at net.sourceforge.cobertura.javancss.parser.JavaParser.ForStatement(JavaParser.java:4286)
	at net.sourceforge.cobertura.javancss.parser.JavaParser.Statement(JavaParser.java:3735)
	[...]
	at net.sourceforge.cobertura.reporting.ComplexityCalculator.getAccumlatedCCNForSource(ComplexityCalculator.java:105)
	at net.sourceforge.cobertura.reporting.ComplexityCalculator.getAccumlatedCCNForSingleFile(ComplexityCalculator.java:143)
	at net.sourceforge.cobertura.reporting.ComplexityCalculator.getCCNForSourceFileNameInternal(ComplexityCalculator.java:234)
	at net.sourceforge.cobertura.reporting.ComplexityCalculator.getCCNForPackageInternal(ComplexityCalculator.java:201)
	at net.sourceforge.cobertura.reporting.ComplexityCalculator.getCCNForProject(ComplexityCalculator.java:167)
	at net.sourceforge.cobertura.reporting.html.HTMLReport.generateTableRowForTotal(HTMLReport.java:603)
  [...]
	at net.sourceforge.cobertura.reporting.Main.main(Main.java:169)
```

**Comment by sg28** [comment]
Hello ,I am having this issue 


[WARN] JavaNCSS got an error while parsing the java file /home/build/jenkins_workspace/bsp/backend/src/main/java/com/equilar/bsp/util/CalcUtil.java
ParseException in STDIN
Last useful checkpoint: "com.equilar.bsp.util.CalcUtil.isMale()"
Encountered " ">" "> "" at line 33, column 27.
Was expecting one of:
    "assert" ...
    "boolean" ...
    "byte" ...
    "char" ...
    "double" ...
    "enum" ...
    "false" ...
    "float" ...
    "int" ...
    "long" ...
    "new" ...
    "null" ...
    "short" ...
    "super" ...
    "this" ...
    "true" ...
    "void" ...
    <INTEGER_LITERAL> ...
    <FLOATING_POINT_LITERAL> ...
    <CHARACTER_LITERAL> ...
    <STRING_LITERAL> ...
    <IDENTIFIER> ...
    "(" ...
    "!" ...
    "~" ...
    "++" ...
    "--" ...
    "+" ...
    "-" ...
[ERROR] net.sourceforge.cobertura.javancss.parser.ParseException: Encountered " ">" "> "" at line 45, column 36.
Was expecting one of:
    "assert" ...
    "boolean" ...
    "byte" ...
    "char" ...
    "double" ...
    "enum" ...
    "false" ...
    "float" ...
    "int" ...
    "long" ...
    "new" ...
    "null" ...
    "short" ...
    "super" ...
    "this" ...
    "true" ...
    "void" ...
    <INTEGER_LITERAL> ...
    <FLOATING_POINT_LITERAL> ...
    <CHARACTER_LITERAL> ...
    <STRING_LITERAL> ...
    <IDENTIFIER> ...
    "(" ...
    "!" ...
    "~" ...
    "++" ...
    "--" ...
    "+" ...
    "-" ...
    
corbertura plugin 2.7 and java 8
I have been looking for some kind of solution or some way to bypass it ,but havn't found a way around .can i have any suggestions (am very new to this thing) this has created a kind of a blocker for our application which is really getting big in size

**Comment by christ66** [comment]
Fixed as part of #266
